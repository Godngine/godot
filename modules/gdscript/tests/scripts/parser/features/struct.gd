var test_struct: Test

struct Test:
    var number = 25
    var string = "hello"
    var arr = [6]
    var dict = {"hi" : "ho"}


func test():
    print(test_struct)
