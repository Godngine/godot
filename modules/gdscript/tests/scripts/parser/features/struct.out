GDTEST_OK
[number: 25, string: "hello", arr: [6], dict: { "hi": "ho" }]
